js/plugins/conviva.js

```diff
@@ -0,0 +1,962 @@
+require("../framework/InitAnalyticsNamespace.js");
+require("../../html5-common/js/utils/utils.js");
+
+/**
+ * @class ConvivaAnalyticsPlugin
+ * @classdesc Conviva SDK plugin that works with the Ooyala Analytics Framework.
+ * @param {object} framework The Analytics Framework instance
+ */
+var ConvivaAnalyticsPlugin = function(framework)
+{
+  var _framework = framework;
+  var name = "conviva";
+  var version = "v1";
+  var id;
+  var _active = true;
```

Comment:
Was pulled from the analytics template. Removed from all the plugins as it was not used.